docs/javascript/v2/features/pip-mode.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: PIP (Picture-in-Picture) Mode
+nav: 3.37
+---
+
+## Overview
+
+PiP Mode lets the user watch the room video in a small window pinned to a corner of the screen while navigating between apps or browsing content on the main screen.
+
+100ms PiP selects top 4 peers with video enabled and render them in the PiP window.
+
+<video loop="true" autoplay="autoPlay" controls="controls" id="vid" muted>
+    <source src="/docs/guides/pip-demo.mp4" type="video/mp4" />
+</video>
+<br />
+
+## Implemenation
+
+### React projects
+
+For implemenation copy this [folder](https://github.com/100mslive/100ms-web/tree/main/src/components/PIP) to your react project.
+
+```jsx
+import { PIP } from '../path-to-above-folder';
```

Comment:
should we add a note that we are handling pinning tiles and those would be prioritised in PiP. if it is not required they can remove this part. it will still work.

Reply:
i think we should add a line about this just so people don't get stuck

Reply:
@Nixon-Samuel can you fill this up and merge this?

Reply:
done.